operator/pkg/manifest/manifest.go

```diff
@@ -0,0 +1,54 @@
+package manifest
+
+import (
+	"fmt"
+	"os"
+
+	operatorv1alpha1 "github.com/kyma-project/istio/operator/api/v1alpha1"
+	istioOperator "istio.io/istio/operator/pkg/apis/istio/v1alpha1"
+	"sigs.k8s.io/yaml"
+)
+
+var (
+	defaultIstioOperatorFile = "default-istio-operator-k3d.yaml"
+)
+
+// Merge applies configuration from IstioCR to the default Kyma Istio Operator file, saves it to a file and returns its path.
+func Merge(istioCR *operatorv1alpha1.Istio) (string, error) {
+	manifest, err := os.ReadFile(defaultIstioOperatorFile)
```

Comment:
Adding manifestDir const might also be an idea
```suggestion
	manifest, err := os.ReadFile("manifests", defaultIstioOperatorFile)
```